src/js/apps/patients/patient/flow/flow_app.js

```diff
@@ -287,6 +287,7 @@ export default SubRouterApp.extend({
         action.trigger('editing', true);
       },
       'stop'() {
+        Radio.trigger('event-router', 'flow', this.flow.id);
```

Comment:
This approach won't work. Think it's causing issues when navigating away from a `/flow/:id/action/:id` URL.

Reply:
even going from one action to another creates a long browser history